### PR TITLE
adding OpenCensus tracing task with W3C headers

### PR DESCRIPTION
**Pull request by verysonglaa**
Fixes https://github.com/istio/istio.io/issues/8166
This PR is based on https://github.com/istio/istio.io/pull/8270

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [x] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

The images in this PR are created with a locally built bookinfo app, the official images still need to be built with https://github.com/istio/istio/pull/27395 included to support forwarding of W3C trace headers.

### CODE REVIEW

File: content/en/docs/tasks/observability/distributed-tracing/opencensusagent/index.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
+---
+title: OpenCensus Agent
+description: Learn how to configure the proxies to send OpenCensus-formatted spans to OpenTelemetry Collector.
+weight: 10
+keywords: [telemetry,tracing,opencensus,opentelemetry,span,port-forwarding]
+aliases:
+    - /docs/tasks/opencensusagent-tracing.html
+owner: istio/wg-policies-and-telemetry-maintainers
+test: yes
```
Comment (ericvn) [inline content/en/docs/tasks/observability/distributed-tracing/opencensusagent/index.md:9]: You have `yes` here, and it does generate the snips.sh, but there are no tests in this PR.

Comment (verysonglaa) [reply content/en/docs/tasks/observability/distributed-tracing/opencensusagent/index.md:9]: will add tests and rewrite installation to use already existing istio samples

File: content/en/docs/tasks/observability/distributed-tracing/opencensusagent/index.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+              service: "opentelemetry-collector.istio-system.svc.cluster.local"
+              port: 55678
+              context:
+              - W3C_TRACE_CONTEXT
+{{< /text >}}
+
+Otherwise, add the equivalent setting to your original `istioctl install` command, for example:
+
+{{< text syntax=bash snip_id=none >}}
+$ istioctl install -f <your-istio-operator-config-file>
```
Comment (ericvn) [inline content/en/docs/tasks/observability/distributed-tracing/opencensusagent/index.md:47]: My confusion at this point is I don't see any new settings on what would be the original command.

Comment (verysonglaa) [reply content/en/docs/tasks/observability/distributed-tracing/opencensusagent/index.md:47]: I remove this part

File: content/en/docs/tasks/observability/distributed-tracing/opencensusagent/index.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+  namespace: istio-system
+spec:
+  tracing:
+  - randomSamplingPercentage: 100.00
+EOF
+{{< /text >}}
+
+### Deploy OpenTelemetry Collector
+
+OpenTelemetry collector supports exporting traces to a [several backends by default](https://github.com/open-telemetry/opentelemetry-collector/blob/master/exporter/README.md#general-information), with extensions that [support more](https://github.com/open-telemetry/opentelemetry-collector-contrib#exporters).
```
[the remark this answers is not in the document]
Comment (verysonglaa) [reply content/en/docs/tasks/observability/distributed-tracing/opencensusagent/index.md:78]: I reworded it to:
OpenTelemetry collector supports exporting traces to [several backends by default](https://github.com/open-telemetry/opentelemetry-collector/blob/master/exporter/README.md#general-information) in the core distribution. Other backends are available in the [contrib distribution](https://github.com/open-telemetry/opentelemetry-collector-contrib) of OpenTelemetry collector.

File: content/en/docs/tasks/observability/distributed-tracing/opencensusagent/index.md [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+By default, Istio's OpenCensus Agent tracing will attempt to read and write 4 types of trace headers:
+
+* B3,
+* gRPC's binary trace header,
+* [W3C Trace Context](https://www.w3.org/TR/trace-context/),
+* and Cloud Trace Context.
+
+If you supply multiple values, the proxy will attempt to read trace headers in the specified order, using the first one that successfully parsed and writing all headers. This permits interoperability between services that use different headers, e.g. one service that propagates B3 headers and one that propagates W3C Trace Context headers can participate in the same trace. In this example we only use W3C Trace Context.
+
+In the default profile the sampling rate is 1% increase it to 100% using the [Telemetry API](/docs/tasks/observability/telemetry/):
```
[the remark this answers is not in the document]
Comment (verysonglaa) [reply content/en/docs/tasks/observability/distributed-tracing/opencensusagent/index.md]: done

File: content/en/docs/tasks/observability/distributed-tracing/opencensusagent/index.md [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+    to generate trace information.
+
+    {{< boilerplate trace-generation >}}
+
+1.  From the left-hand pane of the dashboard, select `productpage.default` from the **Service** drop-down list and click
+    **Find Traces**:
+
+    {{< image link="./istio-tracing-list.png" caption="Tracing Dashboard" >}}
+
+1.  Click on the most recent trace at the top to see the details corresponding to the
```
[the remark this answers is not in the document]
Comment (verysonglaa) [reply content/en/docs/tasks/observability/distributed-tracing/opencensusagent/index.md:205]: done

**Comment by ericvn** [comment]
Thanks for the updates. I know we are waiting on the updated images. Unfortunately, I don't have access to push them anymore as Docker Hub reduced the number of authorized people. Maybe @frankbu can still do that. Then we can merge this.

**Pull request merged**